clair/clair.go

```diff
@@ -58,8 +59,23 @@ func (imageAnalysis ImageAnalysis) ShortName(l v1.Layer) string {
 //Config configure Clair from configFile
 func Config() {
 	uri = fmtURI(viper.GetString("clair.uri"), viper.GetInt("clair.port")) + "/v1"
-	host = viper.GetString("clair.host")
 	healthURI = fmtURI(viper.GetString("clair.uri"), viper.GetInt("clair.healthPort")) + "/health"
 	Report.Path = viper.GetString("clair.report.path")
 	Report.Format = viper.GetString("clair.report.format")
+
+	headersArray := viper.GetStringSlice("clair.headers")
+	for _, header := range(headersArray){
+		splitHeader := strings.Split(header, "=")
+		headers[splitHeader[0]] = splitHeader[1]
+	}
 }
+
+func SetRequestHeaders(request *http.Request) {
+	for name, value := range headers {
+		if strings.EqualFold(name, "host") {
```

Comment:
passing a host header doesnt work. I cant find the link to the docs where I saw this now, but I tried it before doing this and it didnt work. It ignores host headers if you add them as request.Header

I will update the yaml format though